scapy/arch/windows/__init__.py

```diff
@@ -539,7 +540,7 @@ def sndrcv(pks, pkt, timeout = 2, inter = 0, verbose=None, chainCC=0, retry=0, m
                                     if r.answers(sentpkt):
                                         ans.append((sentpkt, r))
                                         if verbose > 1:
-                                            os.write(1, "*")
+                                            sys.stdout.write("*")
```

Comment:
Unless I misunderstand what you mean, a good way to "wrap" `sys.stdout` would probably be to `os.close(1)` and `os.dup2(x, 1)`: that would work regardless of the method used to write to stdout (`sys.stdout.write()`, `os.write(1, [...])` or `print`).